src/components/MainMemo/MemoButton/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as S from './style';
+import * as T from './type';
+
+function MemoButton({
+  variant,
+  type,
+  text,
+  handleClick,
+  activeColor,
+  activeText,
+}: T.MemoButton) {
+  const onClick = () => {
+    handleClick();
+  };
+
+  return (
+    <S.Button
+      type={type}
+      variant={variant}
+      onClick={onClick}
```

Comment:
```suggestion
  return (
    <S.Button
      type={type}
      variant={variant}
      onClick={handleClick}
```

아래 쓴 것과 마찬가지로 여기도 `handleClick` 바로 쓰면 될 것 같네요~!